docs/content_management/rich_text/extend_online_editor.md

```diff
@@ -393,24 +393,65 @@ For a full list of standard buttons, see the RichText module's [configuration fi
 
 ## Add CKEditor plugins
 
-Custom CKEditor plugin must be added to the `extraPlugins` array under the 
-`window.ibexa.richText.CKEditor.extraPlugins` key.
-For this purpose, either in the bundle's `Resources/encore/` folder, 
-or in the `encore` folder in the root directory of your project, 
-create the following `ibexa.richtext.config.manager.js` file:
+A CKEditor plugin is installed locally by using `yarn add ` or `npm install`. and is deployed by committing the `yarn.lock` file.
 
-``` js
+For example, the local installation of [Special characters plugin](https://ckeditor.com/docs/ckeditor5/latest/features/special-characters.html):
+```bash
+yarn add @ckeditor/ckeditor5-special-characters`
+```
+
+CKEditor plugin must be added to the `ibexa.richText.CKEditor.extraPlugins` array.
+For this purpose, create a JavaScript file to import the plugin elements and add them to the array using `ibexa.addConfig`.
+
+For example, `assets/js/special-characters.js`:
+```js
+// The plugin itself
+import SpecialCharacters from '../../node_modules/@ckeditor/ckeditor5-special-characters/src/specialcharacters';
+// The character list that will be used by the plugin
+import SpecialCharactersEssentials from '../../node_modules/@ckeditor/ckeditor5-special-characters/src/specialcharactersessentials';
+ibexa.addConfig('richText.CKEditor.extraPlugins', [ SpecialCharacters, SpecialCharactersEssentials ], true);
```

Comment:
I tried more adventurous character list like the following. But I don't see the real interest into using such advanced example.
Based on [Adding a new category](https://ckeditor.com/docs/ckeditor5/latest/features/special-characters.html#adding-a-new-category) and [Removing categories](https://ckeditor.com/docs/ckeditor5/latest/features/special-characters.html#removing-categories):

```js
import SpecialCharacters from '../../node_modules/@ckeditor/ckeditor5-special-characters/src/specialcharacters';
import SpecialCharactersArrows from '../../node_modules/@ckeditor/ckeditor5-special-characters/src/specialcharactersarrows';

function SpecialCharactersEmoji( editor ) {
    editor.plugins.get( 'SpecialCharacters' ).addItems( 'Emoji', [
        { title: 'smiley face', character: '😊' },
        { title: 'rocket', character: '🚀' },
        { title: 'wind blowing face', character: '🌬️' },
        { title: 'floppy disk', character: '💾' },
        { title: 'heart', character: '❤️' }
    ], { label: 'Emoticons' } );
}

ibexa.addConfig('richText.CKEditor.extraPlugins', [ SpecialCharacters, SpecialCharactersArrows, SpecialCharactersEmoji, ], true);
```

Reply:
I guess it will be a usecase for some, that you can specify a manual list, and not only import from existing lists.

Reply:
Let's keep the doc simple.
Too advanced example also saved in the ticket.